### DISCUSSION TITLE
more options for recurring donations

### DISCUSSION THREAD

**Discussion by PoeBloodstone** [gameserverapp/Platform#513] [Suggestions]
Currently, there is only recurring month-to-month for donations/say a VIP/supporter program, it would be amazing if there was an option for pull every 3, 6, and 12 months so people and server owners have more options on setting up and doing their supporter packs.

**Comment by mvaessen** [comment]
Hi, i recommend doing single-charge payments for long-term subscriptions. This is much easier and reduces the chance of getting disputes.
